src/main/app/guards/featureToggleGuard.ts

```diff
@@ -12,9 +11,12 @@ export class FeatureToggleGuard {
    * @returns {express.RequestHandler} - request handler middleware
    */
   static featureEnabledGuard (feature: string): express.RequestHandler {
-    return GuardFactory.create(() => FeatureToggles.isEnabled(feature), (req: express.Request, res: express.Response): void => {
-      throw new NotFoundError(req.path)
-    })
+    return GuardFactory.create(
+      (res: express.Response) => FeatureToggles.hasAuthorisedFeature(feature, res.locals.claim.features),
```

Comment:
Please add type to ` (res: express.Response)`.

Reply:
I also do not like the change from `FeatureToggles.isEnabled(feature)` to  `FeatureToggles.hasAuthorisedFeature`. It make `featureEnabledGuard` work only with opt-in features while not every feature will require user opt-in.

Separate guard function should be created for admissions and name should clearly state it checks users consent in addition to checking system config.

Reply:
We should have something like:

```
# method definition
class OptInFeatureToggleGuard {
  static featureEnabledGuard (feature: string) { ... }
}

# use case
OptInFeatureToggleGuard.featureEnabledGuard('admissions')
```